packages/bot/src/transformers/message.ts

```diff
@@ -415,6 +419,16 @@ export function transformMessageInteractionMetadata(bot: Bot, payload: DiscordMe
   return bot.transformers.customizers.messageInteractionMetadata(bot, payload, metadata)
 }
 
+export function transformMessageCall(bot: Bot, payload: DiscordMessageCall): MessageCall {
+  const call = {} as MessageCall
+  const props = bot.transformers.desiredProperties.message.call
+
+  if (props.participants && payload.participants.length > 0) call.participants = payload.participants.map((x) => bot.transformers.snowflake(x))
```

Comment:
```suggestion
  if (props.participants && payload.participants) call.participants = payload.participants.map((x) => bot.transformers.snowflake(x))
```
Why not keep the empty array as-is?

Reply:
depends on if we want to give the user an empty array or undefined, for me it is fine just removing the length check, but we do have it in other places

Reply:
Personally I think that if the user desire the participants property, they should get an empty array